gpflow/likelihoods/base.py

```diff
@@ -274,7 +313,7 @@ def _quadrature_dim(self) -> int:
         assert self.latent_dim is not None
         return self.latent_dim
 
-    def _quadrature_log_prob(self, F: TensorType, Y: TensorType) -> tf.Tensor:
+    def _quadrature_log_prob(self, F: TensorType, X: TensorType, Y: TensorType) -> tf.Tensor:
```

Comment:
why this change in order? Is there a way of working around this?

Reply:
> why this change in order? Is there a way of working around this?

This is because this method is used as `self.quadrature.logspace(self._quadrature_log_prop, ...)` and `self.quadrature` operates over the first parameter, so the first parameter has to be `F`.

To work around this we'd have to update all our quadrature stuff to have special-case handling of `X`, which seems like a bad idea.